FB - Overview PHP EOL/EOS (#7371)
ovh/docs: pages/web_cloud/web_hosting/php_eol_eos/guide.en-gb.md

---

> title: "End of sales announcements/PHP Support"

Reviewer: ```suggestion
title: "PHP - End-of-Sale & End-Of-Life announcements"
```

> excerpt: "Find out when your PHP support/sales will end"

Reviewer: ```suggestion
excerpt: "Find out the main end-of-life, end-of-support and end-of-sales dates for PHP versions for OVHcloud Web Hosting plans"
```

updated: 2025-01-17
---

## Objective

> The information contained in this guide concerns the PHP versions used as part of the Web Hosting services, accessible via the Web Hosting network. This document details the end of sales, end of life, and end of support dates for these releases, helping you to effectively plan your transitions and upgrades.

Reviewer: ```suggestion
The information contained in this guide concerns the PHP versions used as part of the Web Hosting services, accessible via the Web Hosting network. This document details the end of sale, end of life, and end of support dates for these releases, helping you to effectively plan your transitions and upgrades.
```

> **Find out about the main PHP versions' end of life, end of support and end of sales dates.**

Reviewer: ```suggestion
**Find out about the main PHP versions' end of life, end of support and end of sale dates.**
```

## Requirements

> - A [OVHcloud web hosting plan](/links/web/hosting).

Reviewer: ```suggestion
- An [OVHcloud Web Hosting plan](/links/web/hosting).
```
